hydra-access-controls/hydra-access-controls.gemspec

```diff
@@ -19,9 +19,9 @@ Gem::Specification.new do |gem|
   gem.required_ruby_version = '>= 1.9.3'
 
   gem.add_dependency 'activesupport', '~> 4.0'
-  gem.add_dependency "active-fedora", '~> 9.13'
+  gem.add_dependency "active-fedora", '~> 10.0.0.beta1'
```

Comment:
I was hoping we'd take the beta part out in a later release anyway, but I can change it now if you mind.